http-api/src/main.rs

```diff
@@ -28,9 +50,9 @@ pub struct Options {
     #[argh(option, default = r#"String::from("base16-ocean.dark")"#)]
     pub theme: String,
 
-    /// disable colored log output (default: false)
-    #[argh(switch)]
-    pub no_color: bool,
```

Comment:
I don't see what for

Reply:
To prevent colors from ending up in the logs.

Reply:
I can re-add it but it's been added so that machine-readable logs aren't polluted with terminal codes.  Now that the machine-readable logging needs are served by the `gcp` feature, there's no longer need for the black and white logs.  So the original need is no more.

Reply:
I'd leave it as it's useful for non-gcp deployments.